### ISSUE TITLE
Spatial referencing of a PNG image; pHYs unsuitable

### ISSUE THREAD

**Issue by svgeesus** [w3c/PNG-spec#14]
Raised by: Adrian Custer
Original date: [12 Jun 2012](https://lists.w3.org/Archives/Public/png-group/2012Jun/0000.html)

> I am working as part of the Open Geospatial Consortium (OGC) on a web
service standard which exchanges PNG datastreams which are cartographic
maps (or tiles thereof). As part of that work, I am having to establish
how a PNG decoder determines the spatial referencing of a PNG image,
both the initial orientation (where I rely on the orientation of the
'reference image') and the various geospatial referencing elements. For
the latter, I claim that PNG provides *no* information but this forces
me to dismiss the 'pHYs' chunk of the standard as descriptive and also,
possibly, to address currently registered extensions such as 'oFFs'.

**Comment by ProgramMax** [comment]
Indeed, the `pHYs` chunk and `oFFs` are not suitable for geospacial information.

This is related to #15. In fact, they're two separate pieces of the original post.
The original post suggests that a new chunk may or may not be needed to represent geospacial information.
Let's determine that in #15 and then update this issue as needed.
